khmer/_khmermodule.cc

```diff
@@ -4370,6 +4586,12 @@ init_khmer(void)
         return;
     }
 
+    khmer_KHLLCounterType.tp_new = khmer_hllcounter_new;
```

Comment:
Hmm.. @camillescott Is there a reason we are defining `tp_new` and `tp_methods` in the khmer module init function instead with the other `PyTypeObject` struct members?